utils/s2n_map.h

```diff
@@ -30,3 +35,9 @@ S2N_RESULT s2n_map_complete(struct s2n_map *map);
 S2N_RESULT s2n_map_unlock(struct s2n_map *map);
 S2N_RESULT s2n_map_lookup(const struct s2n_map *map, struct s2n_blob *key, struct s2n_blob *value, bool *key_found);
 S2N_RESULT s2n_map_free(struct s2n_map *map);
+S2N_RESULT s2n_map_size(struct s2n_map *map, uint32_t *size);
+
+S2N_RESULT s2n_map_iterator_init(struct s2n_map_iterator *iter, const struct s2n_map *map);
+S2N_RESULT s2n_map_iterator_next(struct s2n_map_iterator *iter, struct s2n_blob *value);
+S2N_RESULT s2n_map_iterator_has_next(const struct s2n_map_iterator *iter, bool *has_next);
+S2N_CLEANUP_RESULT s2n_map_iterator_free(struct s2n_map_iterator *iter);
```

Comment:
s2n_map_iterator_free is gone now, right? I'm surprised the compiler didn't flag that as an error